### ISSUE TITLE
how to change batch size for training

### ISSUE THREAD

**Issue by MichaelGMoore** [DeepLabCut/DeepLabCut#737]
Thanks for opening this issue, and thanks for using DeepLabCut (we hope you are enjoying it ;). Please fill out the template completely, including the full "traceback" and input code that you ran to hit this error.

**Your Operating system and DeepLabCut version**

Ubuntu 16.04.6 LTS with Anconda Env DLC-GPU & DeepLabCut2.2b5

Please complete the following information about your system:

OS:  Ubuntu Linux
DeepLabCut Version: 2.2b5
Anaconda env used: DLC-GPU

**Describe the problem**
Edit config.yaml to change batch_size from 8 to 1
Then run: deeplabcut.train_network(path_config_file)
and it shows as batch_size = 8

This is important because I am getting resource allocation error running RTX2080Ti on 1024x768 images.

If I manually edit the pose_cfg.yaml file in the right sub-folder inside dlc-models folder, setting batch_size = 1, then the training can run without error, but it shouldn't have to be done this way. Setting it in the main config.yaml should be enough.

Please place code inside this:

<details><summary>Code output</summary><p>

[Please copy/paste the full terminal output of the error here!!!]

</p></details>

**How to Reproduce the problem**

**Screenshots**

**Additional context**

**Comment by waylonflinn** [comment]
Here's a very good explanation of the relationship between batch size and learning rate. Perhaps this could be used to allow easily setting the batch size for training for the user while automatically adjusting the learning rate.

https://stackoverflow.com/a/66546571/74291
